### PR TITLE
updated spelling for docs generation

### PR DESCRIPTION
**Pull request by benPearce1**

### CODE REVIEW

File: pkg/cmd/login/login.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func NewCmdLogin(f factory.Factory) *cobra.Command {
 		Long:  "Login to your Octopus server using OpenID Connect (OIDC) or an API key. If no arguments are provided then login will be done interactively allowing creation of an API key.",
 		Example: heredoc.Docf(`
 			$ %[1]s login
-			$ %[1]s login --server https://my.octopus.app --service-account-id b1a6f20f-0ec7-4e9a-938e-db800f945b37 --id-token eyJhbGciOiJQUzI1NiIsImtp...
```
Comment (benPearce1) [inline pkg/cmd/login/login.go:60]: Imtp was coming up as a misspelt word

Comment (hnrkndrssn) [reply pkg/cmd/login/login.go:60]: and `NiIs` isn't 🤯 🤷

**Review by hnrkndrssn — APPROVED** [review]
👍

File: pkg/cmd/login/login.go (hunk 1 above)
Comment (benPearce1) [reply pkg/cmd/login/login.go:60]: The captial `I` starts a new word

**Pull request merged**